lib/ddtrace/contrib/action_cable/events/broadcast.rb

```diff
@@ -0,0 +1,45 @@
+require 'ddtrace/contrib/analytics'
+require 'ddtrace/contrib/action_cable/event'
+
+module Datadog
+  module Contrib
+    module ActionCable
+      module Events
+        # Defines instrumentation for 'broadcast.action_cable' event
+        module Broadcast
+          include ActionCable::Event
+
+          EVENT_NAME = 'broadcast.action_cable'.freeze
+
+          module_function
+
+          def event_name
+            self::EVENT_NAME
+          end
+
+          def span_name
+            Ext::SPAN_BROADCAST
+          end
+
+          def span_type
+            Datadog::Ext::AppTypes::CUSTOM
```

Comment:
Should this be considered "http" instead? As it is sending web requests? Or would it be more accurate to say this is a distinct kind of web transmission? In which case we should consider adding a new span type.

Reply:
All `action_cable` spans have been moved to `web`.